README.rst

```diff
@@ -898,6 +898,114 @@ Delete VPN
 
     DELETE /api/v1/controller/vpn/{pk}/
 
+List CA
+^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/pki/ca/
```

Comment:
as shared in the gitter chat, let's follow the URL naming consistently with the other sub apps, so `/api/v1/<module-name>/<resource-name>/` which translates to `/api/v1/controller/ca/`, applies to other occurrences.